qunicorn_core/core/mapper/deployment_mapper.py

```diff
@@ -49,3 +50,25 @@ def dataclass_to_dto(deployment: DeploymentDataclass) -> DeploymentDto:
             "programs": [quantum_program_mapper.dataclass_to_dto(qc) for qc in deployment.programs],
         },
     )
+
+
+def dto_to_response(deployment: DeploymentDto) -> DeploymentResponseDto:
```

Comment:
Vlt wäre ein "Program X in <QISKIT>" oder sowas vergleichbares besser. So wird nicht ganz ersichtlich das es sich tatsächlich um programs und nicht nur Assembler Languages handelt